hi-ml-histopathology/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Using the histopathology models
+
+## Onboarding to Azure
+
+Prio 1, to be completed later
+
+## Creating datasets
```

Comment:
Would we include here a few words on how to define custom dataset classes and data modules (e.g. if user wants to train on new dataset), or is that out of scope for now?

Reply:
Dataset format depends on the models (tiles datasets versus whole slide datasets). I have included a section "how to train on your data" for each of the models.